python/tvm/contrib/cutlass/gen_gemm.py

```diff
@@ -335,6 +347,16 @@ def check_align(self, op_name, M):
             return False
         return True
 
+    def get_default(self, out_dtype):
+        """Return the default kernel for the requested architecture.
+        For now, the default kernel was picked arbitrary.
+        """
+        ops = GENERATOR_FUNC_TABLE[self.sm](out_dtype)
+        default_kernel_name = DEFAULT_KERNELS[self.sm][out_dtype]
```

Comment:
Check whether the SM is listed.

Reply:
Added an additional check in the constructor https://github.com/apache/tvm/blob/6fb9e98090733ed5243a2cca10910cebfab83a76/python/tvm/contrib/cutlass/gen_gemm.py#L339